modules/learn/pages/data/transactions.adoc

```diff
@@ -45,44 +30,78 @@ If two transactions simultaneously attempt to write to the same document, one is
 +
-If, on a single-node cluster, a bucket's replica-assignment was previously greater than zero, and the number is reduced to zero in order to support synchronous writes, a rebalance must be performed prior to any synchronous write.
+NOTE: If using a single node cluster (for example, during development), then note that the default number of replicas for a newly created bucket is 1.
+If left at this default, then all Key-Value writes performed at with durabiltiy will fail with a `DurabilityImpossibleException`.
+In turn this will cause all transactions (which perform all Key-Value writes durably) to fail.
+This setting can be changed via xref:manage:manage-buckets/create-bucket.adoc#couchbase-bucket-settings[GUI] or xref:cli:cbcli/couchbase-cli-bucket-create.adoc#options[command line].
+
+* A Transactional N1QL Query using `AT_PLUS` will be automatically upgraded to `REQUEST_PLUS`.
+
+=== Additional storage use
+
+Transactions make use of the xref:learn:data/durability.adoc[Durability] provided by _synchronous writes_.
+As well as making use of individual documents' extended attributes, transactions also create additional documents in each bucket that they access.
+These include:
+
+*  _Active Transaction Records_.
+Multiple Active Transaction Records can exist per bucket.
+Each of these documents can contain entries for multiple transactions.
+Collectively, these documents record all currently active transactions for the bucket.
+The name of each document is prefixed with `&#95;txn:atr-`.
+
+* _Transaction Client Records_.
+One of these exists per bucket, covering all clients currently accessing the bucket transactionally.
+The document is named `&#95;txn:client-record`.
+
+These documents, which may persist indefinitely, are automatically maintained by Couchbase Server, and should not be modified by any application.
+
 
-[#creating-and-using-transactions]
 == Creating and Using Transactions
-For examples of creating and using transactions on Couchbase Server, see xref:3.0@java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Distributed Transactions from the Java SDK].
+
+For examples of creating and using transactions on Couchbase Server, see:
+
+* xref:cxx-txns::distributed-acid-transactions-from-the-sdk.adoc[Distributed Transactions through the C++ API].
+* xref:dotnet-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Distributed Transactions from the .NET SDK].
+* xref:java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Distributed Transactions from the Java SDK].
+
+NOTE: SDK Transactions Version 1.0.x do not support N1QL operations, but can be used with Couchbase Server 6.6.
+SDK Transactions Version 1.1.x require at least Server 7.0, and will work with N1QL operations.
```

Comment:
We don't have a consistent versioning scheme across all transaction libs so I wouldn't go with this.. Also AFAIK all versions of all transaction libs will be compatible with 6.6.1.
(We should explicitly call out 6.6.1 rather than 6.6 if we need to mention versions - 6.6.0 had a bad bug.)

Reply:
I think we should maybe just avoid mentioning specific transaction library versions at all?  If someone's using Java then they don't need to know that .NET doesn't yet support query.  It just looks bad.

Reply:
Okay

Reply:
Maybe I'm just unfamiliar with the syntax here, but it looks like the NOTE: line is still present and not commented out?  As it looks to be on a newline after the ////